desc/coils.py

```diff
@@ -834,7 +849,7 @@ class FourierPlanarCoil(_Coil, FourierPlanarCurve):
     basis : {'xyz', 'rpz'}
         Coordinate system for center and normal vectors. Default = 'xyz'.
     name : str
-        Name for this coil.
+        name for this coil
```

Comment:
I prefer proper punctuation in our doc strings, I think we do that in most cases

Reply:
we are pretty inconsistent even in this file